django/contrib/staticfiles/storage.py

```diff
@@ -69,7 +69,8 @@ class HashedFilesMixin:
             "*.js",
             (
                 (
-                    r"(?m)(?P<matched>)^(//# (?-i:sourceMappingURL)=(?P<url>.*))$",
+                    r"(?m)(?P<matched>)"
+                    r"^(//# (?-i:sourceMappingURL)=(?P<url>.*?))[ \t]*$",
```

Comment:
Should we allow tabs before `sourceMappingURL`?
```suggestion
                    r"^(//#[ \t](?-i:sourceMappingURL)=(?P<url>.*?))[ \t]*$",
```

Reply:
Please ignore, I missed your comment in the ticket description.